### ISSUE TITLE
Does it support IOS or windows

### ISSUE THREAD

**Issue by mendax1234** [SUPERCILEX/Robot-Scouter#305]
**Is your feature request related to a problem? Please describe.**
l don't have an android phone. l use iPhone and can l use this app.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by SUPERCILEX** [comment]
No, the plan was to support web (and thereby iOS), but I'm no longer actively developing Robot Scouter unfortunately.

**Comment by mendax1234** [comment]
How can it support the web?

**Comment by SUPERCILEX** [comment]
All the data is stored in firebase: someone just needs to build a UI that uses that data.
